swift-sdk/Internal/IterableAPIInternal.swift

```diff
@@ -28,7 +28,8 @@ final class IterableAPIInternal: NSObject, PushTrackerProtocol, AuthProvider {
             
             _email = newValue
             _userId = nil
-            storeEmailAndUserId()
+            
```

Comment:
`authorization = nil` for symmetry

Reply:
I mean `authToken = nil`

Reply:
I can't do that because `authToken`for the new email gets set before this call.

https://github.com/Iterable/swift-sdk/pull/331/files#diff-a003af0114c3087fec8bdff95b1a37c7R139

Reply:
I see you are calling IterableAPI.email from IterableAPI.setEmail... OK.